articles/storage/common/storage-network-security.md

```diff
@@ -288,6 +288,7 @@ When the "Trusted Microsoft Services" exception is enabled, the following servic
 
 |Service|Resource Provider Name|Purpose|
 |:------|:---------------------|:------|
+|Azure Backup|Microsoft.Networking|Perform and continue with scheduled and ad-hoc IaaS VM backups and restores.  [Learn more](https://docs.microsoft.com/en-us/azure/backup/backup-introduction-to-azure-backup).|
```

Comment:
looks like the second column here is wrong - can you add the Resource Provider name for Microsoft Backup here?

Reply:
@AlanSt , FYI